ament_clang_format/package.xml

```diff
@@ -9,6 +9,11 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
```

Comment:
E.g. every non-Python packages would need an explicit `cmake` dependency if we apply the same "rules". Imo both would make it much more effort for users of ROS since it is boilerplate and redundant information. It would be possible to inject this kind of dependency based on the build type. Not sure where in the process this could live best though.